system/jlib/jbuff.cpp

```diff
@@ -174,7 +174,10 @@ MemoryAttr::MemoryAttr(const MemoryAttr & src)
 
 void MemoryAttr::set(size_t _len, const void * _ptr)
 {
-    memcpy(allocate(_len), _ptr, _len);
+    if (_len)
```

Comment:
Does this change the existing behaviour?

Reply:
I've just double checked, and it will behave the same.